Gemfile.lock

```diff
@@ -616,18 +616,18 @@ GEM
     rexml (3.2.4)
     rolify (5.3.0)
     rouge (3.26.0)
-    rspec (3.9.0)
-      rspec-core (~> 3.9.0)
-      rspec-expectations (~> 3.9.0)
-      rspec-mocks (~> 3.9.0)
-    rspec-core (3.9.2)
-      rspec-support (~> 3.9.3)
-    rspec-expectations (3.9.2)
+    rspec (3.10.0)
+      rspec-core (~> 3.10.0)
+      rspec-expectations (~> 3.10.0)
+      rspec-mocks (~> 3.10.0)
+    rspec-core (3.10.0)
+      rspec-support (~> 3.10.0)
```

Comment:
enhancements and bug fixes: https://github.com/rspec/rspec-core/blob/v3.10.0/Changelog.md#3100--2020-10-30